Ansible ssh changes docswork

docschick opened the pull request:
Versioning note for removing "ssh" from "ansible_user" variables in 2.0, which doesn't work in older versions

Commit:
added note about ansible_ssh_* change

added a note like the following to each file hit with unlabled 2.0 changes...
Ansible 2.0 moved away from using ansible_ssh_* variables to accepting
ansible_* variables. If you are using a version of Ansible prior to 2.0,
you should continue using the older style variables (ansible_ssh_*), such
as ansible_ssh_user instead of ansible_user and ansible_ssh_port instead of
ansible_port, which appear in the following content. These shorter variables
are ignored, without warning, in older versions of Ansible.

Commit:
note modified a bit, then reviewed/approved by jmartin

Ansible 2.0 has depricated the “ssh” from ansible_ssh_user,
ansible_ssh_host, and ansible_ssh_port to become ansible_user,
ansible_host, and ansible_port. If you are using a version of
Ansible prior to 2.0, you should continue using the older style
variables (ansible_ssh_*). These shorter variables are ignored,
without warning, in older versions of Ansible.

Commit:
fixed git conflict with double para

bcoca commented:
would it not be easier to just create a cross reference and point at it from each instance?

docschick commented:
Yes, it would. I'd prefer it done using an "include" variable, so that it gets posted in each section but only has to be maintained in one location.  Cross-referencing could be okay, but I'd feel better about including the actual note instead of a link to it and some text telling the user why they need to view this link.   When I wrote the original notes, I had them tailored for each section. After talking with jmartin, we went with a more succinct note that would work for all. It just didn't dawn on me at the moment that repetition like that was less than ideal.   So, to take this approach, where should I put the shared content to be referenced among the various pieces? For Tower, I made a common directory at the same level as the directories for the various manuals. Is there something like this already in place for Core?

amenonsen commented:
depricated → deprecated everywhere.

Commit:
fixed typo, depricated --> deprecated, in all 6 notes

bcoca commented:
We don't have a place like that, but in this case i might put it where we list the variables.

Commit:
removed repetition of note and replaced with an include directive

ansible_ssh_* changes from 1.9 to 2.0, original note made into a separate file
for easier editing, and an include for this new file added to each of the 6 file affected
by this change
